src/App.jsx

```diff
@@ -4,13 +4,17 @@ import Header from './components/Header.jsx/Header';
 import Articles from './components/Articles';
 
 import 'bootstrap/dist/css/bootstrap.min.css';
+import ArticleView from './components/ArticleView';
 
 function App() {
 	return (
 		<>
 			<Header></Header>
 			<Routes>
 				<Route path="/" element={<Articles></Articles>}></Route>
+				<Route
+					path="/article/:id"
+					element={<ArticleView></ArticleView>}></Route>
```

Comment:
This is so minor but you can use 
`<ArticleView/>`
as shorthand for
`<ArticleView></ArticleView>`